src/dune_rules/sub_dirs.ml

```diff
@@ -79,11 +79,7 @@ let status status_by_dir ~dir : Status.Or_ignored.t =
   | Some d -> Status d
 
 let default =
-  let standard_dirs =
-    Predicate_lang.Glob.of_pred (function
-      | "" -> false
-      | s -> s.[0] <> '.' && s.[0] <> '_')
-  in
+  let standard_dirs = Predicate_lang.Glob.of_glob (Glob.of_string "[!._]*") in
```

Comment:
Just checking: does that match single-character directories like `a`? (I'm not sure if `*` matches the empty string in that position)

Reply:
I'll add a unit test for this

Reply:
Indeed it works. See test.